Makefile

```diff
@@ -31,9 +31,18 @@ protoc-docker:
 	docker pull bufbuild/buf:1.14.0
 	docker run --rm --user $$(id -u):$$(id -g) -v $(PWD):/work --tmpfs /.cache -w /work/proto bufbuild/buf:1.14.0 generate -v
 
+.PHONY: metal-api
+metal-api:
```

Comment:
What is this target used for? Isn't the mini-lab-push target sufficient?

Reply:
Seems like a leftover from a different try. Thanks!